### DISCUSSION TITLE
Feature Request - scrutiny

### DISCUSSION THREAD

**Discussion by mitchross** [tteck/Proxmox#376] [Ideas]
Please add https://github.com/AnalogJ/scrutiny

**Comment by AuthorShin** [comment]
Still looking forward for this

**Comment by levvvy** [comment]
Bump, very useful app, it would be nice, to have a script 🙏

**Comment by revilo951** [comment]
Yes! This would be a great addition to your already-awesome arsenal.

**Comment by jpbaril** [comment]
Just stumbled on this issue by searching Google for "proxmox + scrutiny"...
So, yes, that would be great to add that.

**Comment by flopon** [comment]
Yes, very usefull app !

**Comment by navhawk80** [comment]
+1 please
